Content.Server/Explosion/EntitySystems/ExplosionSystem.TileFill.cs

```diff
@@ -88,9 +89,9 @@ public sealed partial class ExplosionSystem
         var spaceAngle = Angle.Zero;
         if (referenceGrid != null)
         {
-            var xform = Transform(Comp<MapGridComponent>(referenceGrid.Value).Owner);
-            spaceMatrix = xform.WorldMatrix;
-            spaceAngle = xform.WorldRotation;
+            var xform = Transform(referenceGrid.Value);
+            spaceMatrix = _transformSystem.GetWorldMatrix(xform);
+            spaceAngle = _transformSystem.GetWorldRotation(xform);
         }
```

Comment:
Use the overload that does both at the same time please, much faster (each one needs to go up the transform hierarchy individually instead of at the same time).